### DISCUSSION TITLE
Chakra V3 & Panda CSS

### DISCUSSION THREAD

**Discussion by Fouadktb** [chakra-ui/chakra-ui#8990] [Q&A] [answered]
Hey Everyone, 

I have existing project where I composed Ark components with Panda css.

I want to move to chakra v3 and remove ark. I just want to know how to use the existing panda recipes with the generated chakra components.

since for example the generated Chakra Component already have size, variant defined, so If I use the recipe and extend the types it'll error.

`
Interface 'ButtonProps' cannot simultaneously extend types 'ButtonProps' and '{ variant?: ConditionalValue<"primary" | "secondary" | "teritary" | "backbutton" | undefined>; size?: ConditionalValue<"sm" | "lg" | "icon" | undefined>; }'.
  Named property 'size' of types 'ButtonProps' and '{ variant?: ConditionalValue<"primary" | "secondary" | "teritary" | "backbutton" | undefined>; size?: ConditionalValue<"sm" | "lg" | "icon" | undefined>; }' are not identical.
`

here is an example
![code](https://github.com/user-attachments/assets/0f8ec59f-ba44-4e46-ab9f-ca3efe583921)

Thanks

**Comment by segunadebayo** [comment, marked as answer]
Chakra v3 uses Emotion recipes. Do not wrap Chakra `Button` in a Panda `cva`.
